packages/nimble-components/src/table-column/base/tests/table-column.fixtures.ts

```diff
@@ -41,20 +43,66 @@ export class TableColumnEmpty extends TableColumn {
     }
 }
 
-export const tableColumnDelegatesClickAndKeydownTag = 'nimble-test-table-column-delegates';
+const configValidity = ['invalidFoo', 'invalidBar'] as const;
+
 /**
- * Simple empty table column with 'click' and 'keydown' event delegation for testing
+ * Column validator used by TableColumnValidationTest
+ */
+export class TestColumnValidator extends ColumnValidator<
+    typeof configValidity
+> {
+    public constructor(columnInternals: ColumnInternals<unknown>) {
+        super(columnInternals, configValidity);
+    }
+
+    public validateFoo(isValid: boolean): void {
+        this.setConditionValue('invalidFoo', !isValid);
+    }
+
+    public validateBar(isValid: boolean): void {
+        this.setConditionValue('invalidBar', !isValid);
+    }
+}
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-test-table-column-validation': TableColumnValidationTest;
+    }
+}
+
+export const tableColumnValidationTestTag = 'nimble-test-table-column-validation';
+/**
+ * Test column type used to verify column config validation
```

Comment:
It's probably worth describing the validation behavior in this comment a bit so that it is clear how the validation works without looking through the code. Something about the column being valid if `foo` and `bar` are both `true` (I think that's the behavior?).

Reply:
Sure